test/test-framework/deploy/olm-catalog/memcached-operator/0.0.4/memcached-operator.v0.0.4.clusterserviceversion.yaml

```diff
@@ -0,0 +1,177 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
This should go in `0.0.3`'s CSV. We aren't updating any field content, just indentation. No need to make a new version/file.

Reply:
👍 I got